src/pages/learn.jsx

```diff
@@ -10,7 +11,23 @@ import organization2 from '../../public/images/partner2.png'
 import roundorca from '../../public/images/roundorca.png'
 import salishsea from '../../public/images/salishsea.png'
 import learnStyles from '../styles/Learn.module.css'
+import {Box, Typography} from "@mui/material"
+import { createTheme,ThemeProvider } from '@mui/material/styles'
 export const learn = () => {
+  const paragraph=createTheme();
+paragraph.typography.p={
+  fontFamily:"Montserrat",
+  fontSize:'1.25rem',
+  fontWeight: '600'
+
+}
+const heading=createTheme();
+heading.typography.h1={
+  fontFamily: 'Montserrat',
+  fontSize: '2.75rem',
+  fontWeight: '600'
+
+}
```

Comment:
Ok I have made the requested changes. I think I have an older version of the project. So should I upload the changes from that branch or create a new branch with the latest version of the project?

Reply:
It's up to you how you want to do it, but what I would do is fetch/pull the latest changes into the main branch, switch to the feature branch, then run `git merge main` and fix the merge conflicts

Reply:
I fetched the latest changes but on my master branch, so there's a conflict now. So I created another local repo and made the required changes. It is the same. Can I upload the changes from a new branch now?

Reply:
That's completely up to you. It would be easiest to keep using this PR (if you're not sure how to deal with merge conflicts [read this](https://www.atlassian.com/git/tutorials/using-branches/merge-conflicts), it's an important skill to learn). But if you'd rather open a new PR that's fine too. You can also force push (`git push -f`) your newest code to this branch which will update this PR.

Reply:
I'll open a new PR as I find it easier as of now. I'll definitely read about merge conflicts and use it in future. Thank you

Reply:
I have made a new PR at the following link:https://github.com/orcasound/orcahome/pull/81

Reply:
Sounds good, go ahead and close this PR then if everything is in the other one now